exporter/ststopologyexporter/exporter_test.go

```diff
@@ -24,6 +24,7 @@ func TestExporter_pushResourcesData(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, 1, len(payload.Topologies))
 		require.Equal(t, 3, len(payload.Topologies[0].Components))
+		require.Equal(t, 2, len(payload.Topologies[0].Relations))
```

Comment:
Would also be good to check, for both component and relation, that the expected attributes are present and others are not (like no API key but there is a service.name)

Reply:
wanted to do only very basic checks here and do the detailed verification in the "topology test".  Makes sense to check for the `sts_api_key`, so I've added that.